flocker/node/functional/docker-images/README

```diff
@@ -0,0 +1 @@
+This directory contains Docker build contexts (and associated Dockerfiles) necessary to build images that are used by the test suite and required to be available to be pulled.
```

Comment:
After modifying these Dockerfiles something something Dockerhub needs to be done. Please document that or link to relevant documentation. If it is big, that can be a follow up issue (but maybe put notes of how you do it on the issue).

This may also affect in-production documentation / tutorials in the future. But that isn't the case now so should probably not be documented.